### DISCUSSION TITLE
HIBP: why just for passwords?

### DISCUSSION THREAD

**Discussion by ledgeri** [keepassxreboot/keepassxc#8590] [Q&A] [answered]
What are the difficulties trying to check the "username" fields for at least emails (must contain @, or similar limitation), while doing the same request from the HIBP service?
I personally would trade an "only emails ca be found"-warning for the mindset of "ohh my new alias is already leaked".
It would be a nice, usefull addition, specially when who reaches the point of PW managers, and HIBP, maybe around the "email aliases" topic too.

**Comment by droidmonkey** [comment, marked as answer]
Username checks is a paid service.

**Reply by ledgeri** [reply]
* *sadface noises* * Thanks...
